### DISCUSSION TITLE
Can ButtonSelector add Icon attributes to different buttons?

### DISCUSSION THREAD

**Discussion by Huoyubai114** [lowdefy/lowdefy#1149] [Questions and Help]
Hi，In the Button, we can set the attribute icon. Can we set different icons for each option in ButtonSelector? How can I achieve it?
    
       - id: pic
        type: Button
        properties:
          icon: 
            name: AccountBookFilled
            size: 21
          shape: square
          ghost: true
          hideTitle: true

**Comment by Gervwyk** [comment]
Hi @Huoyubai114 
This is unfortunately not possible with current features in `ButtonSelector`. Perhaps if can be done by using multiple buttons, with a SetState action to change values?

**Reply by Huoyubai114** [reply]
Thank you for your answer
